packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -217,10 +217,10 @@ describe.each([['petstore.oas2.json'], ['petstore.oas3.json']])('server %s', fil
     // accorging to the schema
 
     const expectedValues = {
-      'x-rate-limit': file === 'petstore.oas3.json' ? 1000 : expect.any(String),
-      'x-stats': file === 'petstore.oas3.json' ? 1500 : expect.any(String),
+      'x-rate-limit': file === 'petstore.oas3.json' ? 1000 : expect.any(Number),
+      'x-stats': file === 'petstore.oas3.json' ? 1500 : expect.any(Number),
       'x-expires-after': expect.any(String),
-      'x-strange-header': file === 'petstore.oas3.json' ? 'string' : '{}',
+      'x-strange-header': 'string',
```

Comment:
When we spoke on slack I meant this should be an empty string, because an empty string is more logical than an empty object.

This should follow the same rules as the response bodies, which are so far not quite behaving as wed expect. 

static: should try the various sources of example, then fall back to type based defaults if required, or null if not required.
dynamic: lorum ipsum or random numbers etc

When we implement static as the default, and have dynamic on the switch, I would expect these headers to follow those same rules. Does that make sense?

Reply:
Sure! 9a80dcb